spec/index.bs

```diff
@@ -11033,6 +11033,7 @@ dictionary GPURenderPassDescriptor
 <script type=idl>
 dictionary GPURenderPassColorAttachment {
     required GPUTextureView view;
+    GPUIntegerCoordinate depthSlice = 0;
```

Comment:
We should make this optional, and then validate that it's provided, if the view is a 3d texture.

Reply:
Currently we set default value for both 2d texture view and 3d texture view, and 1) `depthSlice` can be default setting if using first slice of 3d texture view，2) `depthSlice` cannot be a value other than 0 for non-3d texture view.

If changed, that means `depthSlice` must be provided for 3d texture view, and can be ingroned for non-3d texture view which we don't need to validate it must be 0 again (it's only used in 3d texture view).